ballerina/README.md

```
@@ -2,49 +2,53 @@
-## Quickstart
-
-The fastest path is to generate a typed parser from an EDIFACT or X12 spec using `edi-tools` and call the generated functions from your code.
+## Setup
 
-### Install
-
-The `edi` module is pulled in automatically when you import it. Install the CLI tool:
+The `edi` module is pulled in automatically when you import it. To generate typed parsers from EDI schemas, install the companion CLI tool:
 
 ```bash
 $ bal tool pull edi
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)